### ISSUE TITLE
Add a laser pointer to the collaboration feature

### ISSUE THREAD

**Issue by s0er3n** [zed-industries/zed#14876]
### Check for existing issues

- [X] Completed

### Describe the feature

Sometimes its hard to follow what the other person is talking about so it would be cool if there is a cursor or laser pointer.

**Comment by notpeter** [comment]
We've talked about some ways to better support visual communication, in particular things like "Draw on Video" (where call participants can draw on a hosts screenshare) enhanced mouse cursor and text cursor visibility in Collab sessions.

Thanks for the feedback/ideas.
